MinecraftClient/ChatBot.cs

```diff
@@ -878,48 +899,48 @@ protected bool InteractEntity(int EntityID, int type)
         }
 
         /// <summary>
-        /// Give Creative Mode items into regular/survival Player Inventory
+        /// Interact with an entity
         /// </summary>
-        /// <remarks>(obviously) requires to be in creative mode</remarks>
-        /// <param name="slot">Destination inventory slot</param>
-        /// <param name="itemType">Item type</param>
-        /// <param name="count">Item count</param>
-        /// <returns>TRUE if item given successfully</returns>
-        protected bool CreativeGive(int slot, ItemType itemType, int count)
+        /// <param name="slot"></param>
+        /// <param name="ItemType"></param>
+        /// <param name="count"></param>
+        /// <returns></returns>
+        protected bool CreativeInventoryGive(int slot, ItemType ItemType, int count)
         {
-            return Handler.DoCreativeGive(slot, itemType, count);
+            return Handler.DoCreativeGive(slot, ItemType, count);
```

Comment:
local parameters and variables should start with a lowercase letter, so it's `itemType` here, not `ItemType`.